editor/modes/visual-editor/block.js

```diff
@@ -118,13 +142,23 @@ class VisualEditorBlock extends wp.element.Component {
 								isActive: () => control.isActive( block.attributes )
 							} ) ) } />
 					) : null }
+					{ isSelected && ! isTyping && settings.formatting ? (
+						<Toolbar
+							controls={ settings.formatting.map( ( control ) => ( {
+								...control,
+								onClick: () => this.toggleFormat( control.format ),
+								isActive: () => !! this.state.formats[ control.format ]
```

Comment:
Hmm, reconsidering whether `isActive` really makes sense as a function. We can save this for a separate pull request, but seems like a boolean would be a better substitute